.github/workflows/test_release.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     if: ${{ github.repository == 'dlang/installer' }}
     uses: ./.github/workflows/build_release_template.yml
     with:
-      # Test master for now, could also be safe with stable
-      release_branch: master
+      # Use the same branch/tag, or the targeted branch for PRs
+      release_branch: ${{ github.base_ref || github.ref_name }}
```

Comment:
Analogous to https://github.com/dlang/installer/blob/deb6a77714dff97d11f205bfca576f18c9da9d0b/azure-pipelines.yml#L3